lib/active_merchant/billing/gateways/paypal_express.rb

```diff
@@ -146,7 +146,9 @@ def build_setup_request(action, money, options)
               xml.tag! 'n2:cpp-payflow-color', options[:background_color] unless options[:background_color].blank?
               if options[:allow_guest_checkout]
                 xml.tag! 'n2:SolutionType', 'Sole'
-                xml.tag! 'n2:LandingPage', options[:landing_page] || 'Billing'
+                unless options[:paypal_chooses_landing_page]
```

Comment:
@Krystosterone is this version more readable?

Reply:
I opted for a new parameter as opposed to changing the behaviours of existing params (like touching `:landing_page`) because this could affect other integrations outside of our own

Reply:
I would also have agreed to adding parenthesis to it:
```
xml.tag!('n2:LandingPage', options[:landing_page] || 'Billing') unless options[:paypal_chooses_landing_page]
```